api.js

```diff
@@ -2,7 +2,7 @@ import { TOKEN } from './token.js';
 const axios = require('axios').default;
 
 
-// Handles all get requests, requires a route and a params
+// Handles all GET requests, requires a route and a params
 function handleGetRequests(route, params) {
   let headers;
```

Comment:
I think naming this 'headers' is a misnomer and should be named something like 'options'. Headers are only one property of the options.

Reply:
updated variable names to be "options"